src/services/benefit/benefits.tsx

```
@@ -273,3 +274,31 @@ export const fetchVCJson = async (url: string) => {
+					issuerName: 'dhiway',
+				},
+			},
+			{
+				headers: {
+					'Content-Type': 'application/json',
+				},
+			}
+		);
+
+		return response.data;
+	} catch (error: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)